src/wally-connector.ts

```diff
@@ -267,6 +332,13 @@ class WallyConnector {
     return Object.values(RPCMethodName).indexOf(name as any) > -1;
   }
 
+  /**
+   * @deprecated - see this.request()
+   */
+  public sendAsync(req: any): Promise<any> {
+    return this.request(req);
+  }
```

Comment:
Q: Does the following not work?

```suggestion
  public async sendAsync(req: any) {
    return this.request(req);
  }
```

Reply:
👍